### DISCUSSION TITLE
How to bring bottom sheet to the top

### DISCUSSION THREAD

**Discussion by NandoMB** [gorhom/react-native-bottom-sheet#1600] [Q&A]
There's any way to bring the react-native-bottom-sheet to the top level, to have priority, against the default react native modal? I know that react native's modal is a different component e aways come to the top, but I would like to use this component inside a native modal. The problem is that the bottom sheet is opening behind modal 😞 

Any solution?

**Comment by ManuF17** [comment]
@NandoMB  You can wrap you BottomSheetModal with a FullWindowOverlay from react-native-screens or using react-native-portal, hope this help!

**Reply by dammyluup** [reply]
@ManuF17  am trying to make the BottomSheet go above the bottom tab, how can i implemet that  ?

**Reply by ManuF17** [reply]
You need to wrap your NavigationContainer with the BottomSheetProvider or BottomSheetModalProvider depends on what you are using it

**Reply by dammyluup** [reply]
@ManuF17 thanks 🙏  so much that works..

**Reply by ramakula** [reply]
What about Android? This works only on iOS.

**Reply by claudesortwell** [reply]
> What about Android? This works only on iOS.

Yeah same, android works, ios nope
